src/komal/Assignment_15/BankTransaction.java

```diff
@@ -0,0 +1,173 @@
+package komal.Assignment_15;
+/*Assignment - 15 : 9th April'2021
+	A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+	a) Person can debit or credit amount from respective bank.
+	Hint : Switch case is expected here. [Please watch today's recording 
+	again, in case of confusion.]
+	b) A Method to show total balance [ICICI balance + HDFC balance + BOB 
+	balance + SBI Balance]
+	c) A Method to show total credit, debit operations
+	output : Total credit operation - 7 times
+	Total debit operation - 5 times
+	d) A Method to show individual bank credit operations.
+	output : ICICI bank Credit Operation - 2
+	HDFC bank Credit Operation - 2
+	SBI bank Credit Operation - 1
+	BOB bank Credit Operation - 4
+	e) A Method to show individual bank debit operations.
+	output : ICICI bank Debit Operation - 2
+	HDFC bank Debit Operation - 2
+	SBI*/
+
+public class BankTransaction {
+	
+		double totalBalanceIcici, totalBalanceHdfc, totalBalanceBob, totalBalanceSbi;
+		int debitCountIcici, debitCountHdfc, debitCountBob, debitCountSbi, creditCountIcici, creditCountHdfc,
+				creditCountBob, creditCountSbi;
+		String customerName;
+
+		// Method to set Customer name
+		void setCutomerName(String custName) {
+			customerName = custName;
+			System.out.println(custName + "'s Transaction details as below:");
+		}
+
+		// Method to set initial total balance for respective banks.
+		void setBalance(double balance, String bankName) {
+			switch (bankName.toUpperCase()) {
+			case "ICICI":
+				totalBalanceIcici = balance;
+				break;
+			case "HDFC":
+				totalBalanceHdfc = balance;
+				break;
+			case "BOB":
+				totalBalanceBob = balance;
+				break;
+			case "SBI":
+				totalBalanceSbi = balance;
+				break;
+			default:
+				System.out.println("Invalid bank name!");
+			}
+		}
+
+		// Method to set Bank transaction details.
+		void bankTransaction(String bankName, String transactionType, double amount) {
+			switch (transactionType.toUpperCase()) {			
+				case "DEBIT":
+					switch (bankName.toUpperCase()) {
+						case "ICICI":
+							if (totalBalanceIcici >= amount) {
```

Comment:
instead of writing multi-time same code block you can write debit() method which accept the two arguments and you can simply call that method